tdrs-backend/tdpservice/users/test/test_api/test_set_profile.py

```diff
@@ -27,6 +27,7 @@ def test_set_profile_data(api_client, user):
     )
     assert response.status_code == status.HTTP_200_OK
     assert response.data == {
+        "id": user.id,
```

Comment:
With the `id` added to the related serializer as noted above, all these tests need it too.